service/src/test/java/com/sedliarov/learningtable/integration/StudentControllerIntegrationTests.java

```diff
@@ -35,44 +43,40 @@ public class StudentControllerIntegrationTests extends RestIntegrationTestBase {
 
   @Test
   void testGetStudentById() {
-
     // given
     Student studentForSave = StudentFixture.createEntity();
     Student savedStudent = studentRepository.save(studentForSave);
-    StudentDto studentMapper = mapper.entityToDto(savedStudent);
+    StudentDto expectedStudent = mapper.entityToDto(savedStudent);
 
     // when
-    ResponseEntity<StudentDto> student =
+    ResponseEntity<StudentDto> response =
         exchangeGetWithoutAuth(STUDENTS_URL + savedStudent.getStudentId(), StudentDto.class);
 
     // then
-    assertThat(student.getStatusCode()).isEqualTo(HttpStatus.OK);
-    assertThat(student.getBody()).isEqualTo(studentMapper);
+    assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+    assertThat(response.getBody()).isEqualTo(expectedStudent);
   }
 
   @Test
-  void negativeTestGetStudentById() {
-
+  void testGetStudentByIdIfUserNotExist() {
     // given
-    Student newStudent = StudentFixture.createEntityWithId(UUID.fromString("3e1e6d16-451b-4748-b6a0-8f4a84a0a53a"));
-    Student savedStudent = studentRepository.save(newStudent);
-    StudentDto studentMapper = mapper.entityToDto(newStudent);
+    Student newStudent = StudentFixture.createEntity();
 
     // when
-    ResponseEntity<StudentDto> student =
-        exchangeGetWithoutAuth(STUDENTS_URL + savedStudent.getStudentId(), StudentDto.class);
+    ResponseEntity<StudentDto> response =
+        exchangeGetWithoutAuth(STUDENTS_URL + newStudent.getStudentId(), StudentDto.class);
```

Comment:
My suggestion for it, delete given block and for URL path use static prepared not existing UUIID

Reply:
Done

Reply:
Done